sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/reflection/ParameterExtractor.scala

```diff
@@ -78,7 +78,7 @@ object ParameterExtractors {
     override def extract(context: DynamicMessageContext): T = {
       (required, field.isRepeated || context.message.hasField(field)) match {
         case (_, true)      => deserialize(context.message.getField(field))
-        case (true, false)  => throw BadRequestException(s"Message missing required field: ${field.getName}")
+        case (true, false)  => throw BadRequestException(s"Message is missing required field: ${field.getName}")
```

Comment:
Do we know if it is a path or query parameter here and can we improve the error message to describe that, or some step further out, "message is missing required field" are a bit misleading since the user does not know anything about us encoding the path hand query params as message fields internally.

Reply:
True, I don't like it, it was a copy-paste from some other place. And yes, we know that currently, this applies only to request parameters. Changed.